### ISSUE TITLE
[Bug] opening attribute table in empty mapset freezes GUI

### ISSUE THREAD

**Issue by cmbarton** [OSGeo/grass#3078]
**Describe the bug**
This is a weird bug but it could be a confusing problem for many. If you try to open the attribute table manager from an empty mapset, it freezes the GUI with a 'loading attribute table' message.  

**To Reproduce**
1. Switch to the sc_spm_08_grass7 demo location
2. Create a new mapset ("new") in the data manager
3. Switch to the mapset "new"
4. Make sure that "new" has access to PERMANENT (g.mapsets)
5. Add the firestations vector map from PERMANENT to the layer manager
6. Try to open the attribute table manager window

**Expected behavior**
Instead of the table manager window, there is a 'loading attribute table' message displayed that never goes away. 

**System description (please complete the following information):**

- Operating System: MacOS 13.4.1 (ARM)
- GRASS GIS version: 8.3.0

**Work Around
Copy a vector map to the empty mapset. Then everything works fine and the table manager opens. For some reason, not having the support folders/files for vectors in the current mapset is probably what is causing the freezing. But it shouldn't freeze like that. 

If you then delete the vector you've copied into the empty mapset and try to open the attribute table from another mapset, it works fine. So it definitely has something to do with setting up the support for vectors. Either this needs to happen when mapsets are created or the table manager needs to ignore the lack of vector support when opening a table of a vector in a different mapset.

**Comment by petrasovaa** [comment]
I would expect an error in the GUI console, could you check for it?

**Comment by tmszi** [comment]
I confirm this reported behavior:

Error message is:

```
Traceback (most recent call last):
  File
"/usr/lib64/grass84/gui/wxpython/main_window/frame.py", line
1949, in OnShowAttributeTable

dbmanager = AttributeManager(
  File "/usr/lib64/grass84/gui/wxpython/dbmgr/manager.py",
line 141, in __init__

self.CreateDbMgrPage(parent=self, pageName="manageLayer")
  File "/usr/lib64/grass84/gui/wxpython/dbmgr/base.py", line
842, in CreateDbMgrPage

self.pages[pageName] = DbMgrLayersPage(
  File "/usr/lib64/grass84/gui/wxpython/dbmgr/base.py", line
2868, in __init__

self.manageLayerBook = LayerBook(
  File "/usr/lib64/grass84/gui/wxpython/dbmgr/base.py", line
3100, in __init__

self.defaultConnect["driver"],
self.defaultConnect["database"]
KeyError
:
'driver'
```

**Comment by cmbarton** [comment]
I get the same error

**Comment by petrasovaa** [comment]
This is partially addressed by #3086 in the sense that the db is initialized when the mapset is created in GUI, so this problem is unlikely to happen.
